manifests/service.pp

```diff
@@ -13,7 +13,7 @@
     mode    => '0755',
   }
 
-  if $confluence::manage_service {
+  if str2bool($confluence::manage_service) {
     service { 'confluence':
```

Comment:
I seem to recall an issue where scalar data coming in from Hiera is always a string so str2bool is required? Maybe that got fixed in later versions. I'm going to play it safe and leave str2bool for now. Sometimes modernization is not all it's cracked up to be;-)